linkerd/app/core/src/svc.rs

```diff
@@ -180,6 +180,13 @@ impl<S> Stack<S> {
         self.push(stack::new_service::FromMakeServiceLayer::default())
     }
 
+    pub fn into_make<T>(self) -> Stack<stack::new_service::IntoMakeService<S>>
```

Comment:
nit/TIOLI: the `NewService` conversion layer is `into_new_service`, should this be `into_make_service` rather than `into_make`?

Reply:
There's already `NewService::into_make_service`, and I was hoping to miss that ambiguity (i.e. whether you're getting a `Stack<IntoMakeService<S>>` vs `IntoMakeService<Stack<S>>`.

We could drop the NewService impl on Stack, I guess?